scripts/test-data/profile-l4-x1.yaml

```diff
@@ -39,12 +48,25 @@ evaluate:
     judge_model: ~/.cache/instructlab/models/instructlab/granite-7b-lab
     max_workers: auto
     # Directory where evaluation results are stored
-    output_dir: ~/.local/share/instructlab/internal/eval_data
+    output_dir: ~/.local/share/instructlab/internal/eval_data/mt_bench
   # Settings to run MT-Bench against a branch of taxonomy containing
   # custom skills/knowledge used for training
   mt_bench_branch:
     # Path to where base taxonomy is stored
     taxonomy_path: ~/.local/share/instructlab/taxonomy
+    # Directory where model to be used as judge is stored
+    judge_model: ~/.cache/instructlab/models/instructlab/granite-7b-lab
+    # Directory where evaluation results are stored
+    output_dir: ~/.local/share/instructlab/internal/eval_data/mt_bench_branch
+  # System prompt for model getting responses during DK-Bench.
+  system_prompt: You are an advanced AI assistant designed to provide precise and
+    accurate information. Your primary goal is to answer queries with the most up-to-date
+    and factual information available. Focus on delivering clear, concise, and correct
+    responses. If you're uncertain about any aspect of the query, state your level
+    of confidence and provide the most accurate information you can. Your responses
+    should prioritize accuracy over all other considerations.
+  # Temperature for model getting responses during DK-Bench.
+  temperature: 0.0
```

Comment:
these aren't under the dk_bench section, so is this setting `system_prompt` and `temperature` for benchmark models other than dk-bench?

Reply:
same q applies to the other test-data files too

Reply:
@alinaryan Great catch! Both `temperature` and `system_prompt` are settings which are common to all benchmarks, since they control how the student model generates information. That said, @alimaredia I would avoid setting `system_prompt` explicitly in these files and would instead allow ilab to figure it out internally. This should be used as more of an override setting when users are trying out different ideas.

Reply:
> same q applies to the other test-data files too

We'll be bringing in more Ragas-based metrics that will also be reading in from the test-data files.